src/mlpack/methods/reinforcement_learning/async_learning_impl.hpp

```diff
@@ -89,42 +89,50 @@ void AsyncLearning<
   omp_lock_t tasksLock;
   omp_init_lock(&tasksLock);
 
-  #pragma omp parallel shared(stop, workers, tasks, learningNetwork, \
+  /**
+   * Compute the number of threads for the for-loop. In general, we should use
+   * OpenMP task rather than for-loop, here we do so to be compatible with some
+   * compiler.
```

Comment:
I guess, we could be more specific here, e.g. we can switch to OpenMP task once MSVC supports OpenMP 3.0; might help us later to find that code block again.